[2025-04-16T19:42:14.929+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T19:42:14.972+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-04-16T19:42:13.847072+00:00 [queued]>
[2025-04-16T19:42:14.992+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-04-16T19:42:13.847072+00:00 [queued]>
[2025-04-16T19:42:14.994+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-16T19:42:15.018+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): TestConnectionDB> on 2025-04-16 19:42:13.847072+00:00
[2025-04-16T19:42:15.026+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=276) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-16T19:42:15.027+0000] {standard_task_runner.py:72} INFO - Started process 278 to run task
[2025-04-16T19:42:15.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'BinanceCryptoAPI', 'TestConnectionDB', 'manual__2025-04-16T19:42:13.847072+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/ETL/BinanceETL.py', '--cfg-path', '/tmp/tmpbufv2z9p']
[2025-04-16T19:42:15.031+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask TestConnectionDB
[2025-04-16T19:42:15.391+0000] {task_command.py:467} INFO - Running <TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-04-16T19:42:13.847072+00:00 [running]> on host 2903c312f900
[2025-04-16T19:42:15.627+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BinanceCryptoAPI' AIRFLOW_CTX_TASK_ID='TestConnectionDB' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T19:42:13.847072+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T19:42:13.847072+00:00'
[2025-04-16T19:42:15.628+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-16T19:42:15.668+0000] {base.py:84} INFO - Retrieving connection 'MY_DB'
[2025-04-16T19:42:15.696+0000] {logging_mixin.py:190} INFO - Connection to database successful
[2025-04-16T19:42:15.722+0000] {sql.py:511} INFO - Running statement: SELECT * FROM cryptoInfo, parameters: None
[2025-04-16T19:42:15.726+0000] {sql.py:520} INFO - Rows affected: 0
[2025-04-16T19:42:15.727+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-16T19:42:15.765+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-16T19:42:15.766+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=BinanceCryptoAPI, task_id=TestConnectionDB, run_id=manual__2025-04-16T19:42:13.847072+00:00, execution_date=20250416T194213, start_date=20250416T194214, end_date=20250416T194215
[2025-04-16T19:42:15.928+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T19:42:16.160+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-16T19:42:16.195+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
